### ISSUE TITLE
Info: How to install (balena image - Windows)

### ISSUE THREAD

**Issue by dronecz** [CCHS-Melbourne/openpnp-kiosk#1]
I had bit of struggle to make it work so here is how I did it: 

1) Follow first 4 steps from [Quickstart](https://github.com/CCHS-Melbourne/openpnp-kiosk#quickstart) for balena deploy 
2) Install or download ZIP file with [cli](https://github.com/balena-io/balena-cli/blob/master/INSTALL.md). If you unZIP it from ZIP file, do not forget to add` $PATH` variable to this folder (if you move it somewhere else, then to this location). 
3) Clone or download this (or some other) project. If you download ZIP file, you have to extract it to some folder. 
4) Open command prompt or Powershell window (best as administrator) in location where is project  which you want to upload to balena. TIP: I recommend right-click + holding Shift at destination folder which you want to open in cmd/Powershell window and select "Open cmd/Powershell  window here"
5) type "`balen login`" and choose how you want to login (more help [here](https://github.com/balena-os/balena-engine-cli/blob/master/docs/reference/commandline/login.md)). I did select "Web authorization" which redirected me to webpage where I selected login by Github.
6) type ``balena push`` name_of_your_project/app `--dockerfile .\Dockerfile.raspberrypi3-64`.  I did it in Powershell window, which behave more like linux now, so I´m not sure if this `.\` would not be just `\` in command prompt. 

After that, balena-cli uploaded project to the cloud and from there it was downloaded to ma Raspberry Pi 4. 

I can not see app at my monitor so I will make issue for that.

**Comment by brainstorm** [comment]
You are almost there!

```shell
git clone https://github.com/CCHS-Melbourne/openpnp-kiosk && cd openpnp-kiosk && balena push <APP>
```

Where the `<APP>` must be created via the web first, like:

![Skärmavbild 2020-05-27 kl  10 44 30](https://user-images.githubusercontent.com/175587/82963780-1376f600-a007-11ea-8dfd-9c05d501b9a6.png)

**Comment by brainstorm** [comment]
Solved on issue #2, thanks! ;)
